### DISCUSSION TITLE
Where is the official cmd to install neovim nightly via homebrew on linux/mac-os?

### DISCUSSION THREAD

**Discussion by hopezh** [neovim/neovim#25740] [Q&A] [answered]
May I ask where I can find the official cmd to install `neovim nightly` via `homebrew` on `linux/mac-os` in `neovim wiki`? 

Are they the following?
1. install neovim nightly
`brew install --HEAD neovim
`
3. update/upgrade neovim nightly
`brew upgrade --fetch-HEAD neovim
`

Homework I did:
- https://github.com/neovim/neovim/discussions/25412
- https://github.com/neovim/neovim/discussions/25413
- https://github.com/neovim/neovim/discussions/24954

**Comment by clason** [comment, marked as answer]
There is none; nightly builds via homebrew are not supported. Either build from source or download the nightly build from our release page.

**Reply by hopezh** [reply]
Noted with thanks.
